.github/dependabot.yml

```diff
@@ -0,0 +1,58 @@
+version: 2
```

Comment:
Let's make sure the version is the most up-to-date version

Reply:
Documentation says that this is always 2 for now: https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#about-the-dependabotyml-file